spk/gateone/src/installer.sh

```diff
@@ -35,13 +36,16 @@ postinst ()
     adduser -h ${INSTALL_DIR}/var -g "${DNAME} User" -G ${GROUP} -s /bin/sh -S -D ${USER}
 
     # Create a Python virtualenv
-    ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > /dev/null
+    ${VIRTUALENV} --system-site-packages ${APP_ENV} > /dev/null
 
     # Install the bundle
-    ${INSTALL_DIR}/env/bin/pip install --no-index -U ${INSTALL_DIR}/share/requirements.pybundle > /dev/null
+    ${APP_ENV}/bin/pip install --no-index -U ${INSTALL_DIR}/share/requirements.pybundle > /dev/null
```

Comment:
No problem, I was doing that for convenience for myself, but I completely understand keeping things consistent.